src/workshop/complete/NewTimer.svelte

```diff
@@ -20,11 +20,8 @@
 
   $: ({ duration } = $state.context);
 
-  // TODO There is no focus on first page load (SSR?)
-  async function focus(element) {
-    await tick();
-    element.focus();
-  }
+  let inputEl;
```

Comment:
Can you initialize this like `let inputEl = null;`
Probably has no sense, I just like to be explicit.

Reply:
Committed this.